articles/understanding-static-dynamic-routing/index.md

```diff
@@ -59,7 +59,7 @@ In the image above, a device in sub-network-A want to reach another in sub-netwo
   For a protocol that uses bandwidth capacity as its metric, the protocol determines the bandwidth capacity of all possible routes to the destination subnet, and the route with higher bandwidth capacity is considered the best path and added to the routing table.
 Because metrics can depend on single or many characteristics, Enhanced Interior Gateway Routing Protocol (EIGRP) uses a composite of bandwidth and delay to calculate its metric.
 ![Bandwidth](/engineering-education/understandinding-dynamic-and-static-routing/bandwidth.jpeg)
-[Image source](AUthor)
+Image source: AUthor
```

Comment:
@raboba2re, "Image source: Author"- I don't get its importance in this article, even if it were to be a link, it won't work! Just do away with all of them in this article!

Reply:
OK Sir